src/vsr/replica_test.zig

```diff
@@ -498,8 +498,10 @@ test "Cluster: repair: view-change, new-primary lagging behind checkpoint, forfe
     // Thanks to the new primary, the lagging backup is able to catch up to the latest
     // checkpoint/commit.
     try expectEqual(b1.role(), .backup);
-    try expectEqual(b1.commit(), checkpoint_1_trigger);
+    try expectEqual(b1.commit(), checkpoint_1_trigger + 1);
```

Comment:
Heh, turns out we _were_ testing this already!

Reply:
I am wondering if its worth to add something like

```zig
    try t.expect(snap(@src(),
        \\ 0   / .       3V  23/_27/_27C   0:_28Jo  0/_0J!   0:_31Wo <__0:__0>   49119Gf  0G!   0G?   1/4Pp  0/3Rq
        \\ 1   /  .      1V  23/_28/_28C   0:_28Jo  0/_0J!   0:_31Wo <__0:__0>   49118Gf  0G!   0G?   0/4Pp  0/3Rq
        \\ 2   \   .     3V  23/_27/_27C   0:_27Jo  0/_0J!   0:_31Wo <__0:__0>   49119Gf  0G!   0G?
        \\
    ));
```

to our replica tests, so that surprising cluster states pop out more..

Reply:
:thinking: The VOPR output format prints so much information that it may be hard to spot which parts are important/relevant to the test. Maybe we could wildcard `____` components that the test is unconcerned with, though then it wouldn't be a snap test.

Reply:
>though then it wouldn't be a snap test.

It would! snaps's support ignores:

https://github.com/tigerbeetle/tigerbeetle/blob/6c2cfd7262dba32f2d1f95e1d29a6e3abb6fdc8b/src/integration_tests.zig#L101

(should have called those `snip`s though)